src/pages/billing/CreateAzureBillingProjectModal.test.js

```diff
@@ -13,14 +13,12 @@ import { v4 as uuid } from 'uuid'
 
 
 jest.mock('src/components/Modal', () => {
-  const { div, h } = jest.requireActual('react-hyperscript-helpers')
+  const { h } = jest.requireActual('react-hyperscript-helpers')
   const originalModule = jest.requireActual('src/components/Modal')
   return {
     ...originalModule,
     __esModule: true,
-    default: props => div({ id: 'modal-root' }, [
-      h(originalModule.default, { onAfterOpen: jest.fn(), ...props })
-    ])
+    default: props => h(originalModule.default, { onAfterOpen: jest.fn(), ...props })
```

Comment:
Without this mock, `yarn test` passes but this test fails when I run with `CI=true yarn test`.

Reply:
I think the issue here is that onAfterOpen has logic that requires screen/element dimension measuring services, which are not available in jest environment.

Reply:
Other unit tests can render a Modal without this though.

This test involves a confirmation modal:
https://github.com/DataBiosphere/terra-ui/blob/b7810bde76e140f8c9ca7500ea78b96d237fbbc7/src/components/data/data-table-versions.test.js#L116-L129

Reply:
Aha! `onAfterOpen` isn't the issue.

The tests are passing with this:
```js
jest.mock('src/components/Modal', () => {
  const { h } = jest.requireActual('react-hyperscript-helpers')
  const originalModule = jest.requireActual('src/components/Modal')
  return {
    ...originalModule,
    __esModule: true,
    default: props => h(originalModule.default, { ...props })
  }
})
```

But fail with this (note the props):
```js
jest.mock('src/components/Modal', () => {
  const { h } = jest.requireActual('react-hyperscript-helpers')
  const originalModule = jest.requireActual('src/components/Modal')
  return {
    ...originalModule,
    __esModule: true,
    default: props => h(originalModule.default, props)
  }
})
```

Still don't understand what's going on.

Reply:
Well... now I removed the mock and it works. So I guess I suspect some build caching issue on my machine.

Reply:
Ugh... it seems that this test is flaky. You can see a varying number of failures in the jobs on https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui?branch=modal-root-in-tests.